gnovm/stdlibs/math/overflow/overflow.gno

```diff
@@ -98,8 +149,27 @@ func Quo(a, b int) (int, int, bool) {
 	return int(q32), int(r32), ok
 }
 
+// UQuo returns the quotient, remainder and ok result indicating whether the operation is safe.
+func UQuo(a, b uint) (uint, uint, bool) {
+	if _is64Bit() {
+		uq64, ur64, ok := UQuo64(uint64(a), uint64(b))
+		return uint(uq64), uint(ur64), ok
+	}
+	uq32, ur32, ok := UQuo32(uint32(a), uint32(b))
+	return uint(uq32), uint(ur32), ok
+}
+
 /************* Panic versions for int ****************/
 
+// Absp returns the absolute value, panicking on overflow
+func Absp(x int) int {
+	r, ok := Abs(x)
+	if !ok {
+		panic("absolute value overflow")
+	}
+	return r
+}
```

Comment:
I did the same thing that was done originally by @jaekwon, as for functions such (Addp, Subp...), so If it's the new norm, I'll have to modify even Jae functions. Can you or someone else confirm this?

Reply:
In my opinion, since that is the latest guide recommended by gno and it's the main repository, I think it's appropriate to follow the document first.

I've seen related conversations before, but I didn't find the specific PR/issue.

cc: @moul @leohhhn